src/components/About/TeamCard.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import defaultImage from "../../assets/team.svg";
+import teamMembers from "../../service/about.json";
+
+function TeamCard() {
+  const teamMember = teamMembers.map((member) => (
+    <div
+      className={`mb-10 sm:flex sm:items-center sm:justify-evenly ${
+        member.reversed ? "flex-row-reverse" : "flex-row"
+      }`}
+    >
+      <div className="flex justify-center items-center flex-col">
+        <img
+          src={member.image || defaultImage}
+          alt={member.name}
+          className="w-40"
+        />
+        <h3 className="text-xl font-medium mb-2">
+          {member.name} - {member.position}
+        </h3>
+      </div>
+      <p>{member.description}</p>
+    </div>
+  ));
+
+  return <>{teamMember}</>;
+}
+
+export default TeamCard;
```

Comment:
How about you make this like 
```
const TeamMember = ({image, name, stuff}) => {
 return (
  <div>
    <image src="" />
    <h1> {Name} </h1>
<p>{stuff} </p>
</div
)
}
```

Then in About.jsx page you map through those and pass the data.

Reply:
Another question please. After solving conflicts there are two `About.jsx`. One is in the `pages` and the other is in the `components`. Should I copy the content of the `component/About.jsx` and add it inside `pages/About.jsx` then remove the `About.jsx` inside `components`?